parm/postxconfig-NT-fv3lam.txt

```diff
@@ -9785,6 +9785,43 @@ isothermal
 ?
 ?
 ?
+1006
```

Comment:
@JesseMeng-NOAA The new variable should be incorporated in one RRFS dataset only. You could revert the changes in PRSLEVE dataset.

Reply:
Done